app-frontend/src/app/components/projects/projectItem/projectItem.controller.js

```diff
@@ -1,18 +1,21 @@
 import projectPlaceholder from '../../../../assets/images/transparent.svg';
 
 export default class ProjectItemController {
-    constructor($scope, $state, $attrs, $log, projectService, mapService, mapUtilsService,
-                authService, modalService) {
+    constructor($scope, $state, $attrs, $log,
```

Comment:
@alkamin I ended up using `$rootScope.autoInject(this, arguments);` instead since `$scope` did not seem to work, which makes sense since `$scope` here I think is the component's isolated scope 😄

Reply:
Very interesting! 🤔 Apparently I haven't actually tried it on a component yet. Nice job!